packages/yew/src/server_renderer.rs

```diff
@@ -111,6 +104,35 @@ where
                 .await;
         })
     }
+
+    // The duplicate implementation below is to selectively suppress clippy lints.
```

Comment:
Worth linking to the tracing issue here https://github.com/tokio-rs/tracing/issues/2503